resources/public/api/conf/1.0/benefits_list.yaml

```diff
@@ -7,44 +7,15 @@ description: |
   Any HMRC held benefit amounts that have been amended, will result in a duplicate benefit with the same benefit ID, within the `customerAddedStateBenefits` array.
   
   ### Test data
-  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
-  <table>
-      <thead>
-          <tr>
-              <th>Header Value (Gov-Test-Scenario)</th>
-              <th>Scenario</th>
-          </tr>
-      </thead>
-      <tbody>
-          <tr>
-            <td><p>N/A - DEFAULT</p></td>
-            <td><p>Simulates a list of all state benefit types.</p></td>
-          </tr>
-          <tr>
-              <td><p>N/A - DEFAULT<br>using <strong>benefitId</strong> parameter</p></td>
-              <td><p>Simulates a single HMRC held state benefit.</p></td>
-          </tr>
-          <tr>
-              <td><p>HMRC_AMENDED_AMOUNTS</p></td>
-              <td><p>Simulates the response where a HMRC held state benefit amounts have been amended.</p></td>
-          </tr>
-          <tr>
-              <td><p>NOT_FOUND</p></td>
-              <td><p>Simulates the scenario where no data is found.</p></td>
-          </tr>
-          <tr>
-              <td><p>DYNAMIC</p></td>
-              <td><p>The following response parameters will be based on the data submitted in the request: 
-                <ul>
-                  <li>SubmittedOn</li>
-                  <li>BenefitID</li>
-                  <li>startDate</li>
-                  <li>endDate</li>
-              </ul>
-          </p></td>
-          </tr>
-      </tbody>
-  </table>
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario)    | Scenario                                                                                                                                  |
+  |-------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+  | N/A - DEFAULT                       | Simulates a list of all state benefit types.                                                                                              |
+  | N/A - DEFAULT<br>using **benefitId** parameter | Simulates a single HMRC held state benefit.                                                                                    |
+  | HMRC_AMENDED_AMOUNTS                | Simulates the response where HMRC held state benefit amounts have been amended.                                                           |
+  | NOT_FOUND                           | Simulates the scenario where no data is found.                                                                                            |
+  | DYNAMIC                             | The following response parameters will be based on the data submitted in the request:<br>SubmittedOn<br>BenefitID<br>startDate<br>endDate.|
```

Comment:
Before each of the items that used to be with the <li> tags, can we add ` • ` so that they're bullet pointed